specification/_spec_utils/Stringified.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/**
+ * Some APIs will return values such as numbers also as a string (notably epoch timestamps). This behavior
+ * is used to capture this behavior while keeping the semantics of the field type.
+ *
+ * Depending on the target language, code generators can keep the union or remove it and leniently parse
+ * strings to the target type.
+ */
+export type Stringified<T> = T | string
```

Comment:
Why add this here and not in [`_spec_utils/behaviors.ts`](https://github.com/elastic/elasticsearch-specification/blob/main/specification/_spec_utils/behaviors.ts)?
Also, if this is a behavior, you should add the `@behavior` tag.

Reply:
I initially considered it as a behavior, but it's not. Behaviors are interfaces whose TypeScript definition is essentially empty and are meant to capture things we cannot easily represent in TypeScript.

In this case, it's just a regular type, that code generators may or may not consider in a special way.